app/graphql/types/queries/user_queries.rb

```diff
@@ -0,0 +1,16 @@
+module Queries
+  class User < Queries::BaseQuery
+    def allUsers
```

Comment:
could you explain the difference between putting this in types/queries/user_queries.rb and having a super similar one in types/query_type?

Reply:
This is an on going experiment to put queries that are related in their own file for better organization. right now all queries live in query_type file which will cause merge conflicts